types/heredatalens/index.d.ts

```diff
@@ -618,7 +618,7 @@ declare namespace H {
             /**
              * A factory method for data-driven icons. The method allows you to build an icon from SVG markup or JsonML object. Provides caching of icons with the same markup.
              * @param svg {string | Array} - SVG presented as markup or JsonML Array
-             * @param options {H.map.Icon.Options} - Icon options (eg size and anchor). Note that the default anchor is in the middle.
+             * @param options {H.map.Icon.Options=} - Icon options (eg size and anchor). Note that the default anchor is in the middle.
```

Comment:
Pedantically, I would say that jsdoc in typescript should not include types because they are not checked. And unchecked comments are *always* wrong. :)